cmd/shortener/server/flags.go

```diff
@@ -0,0 +1,28 @@
+package server
+
+import (
+	"flag"
+	"fmt"
+	"github.com/Taboon/urlshortner/cmd/shortener/config"
```

Comment:
Please sort imports

Reply:
Sorted the imports in the entire project